tests/integration/synapseclient/models/async/test_entityview_async.py

```diff
@@ -0,0 +1,1202 @@
+
+        # THEN the entityview should be created
+        assert entityview.id is not None
+
+        # AND I can retrieve that entityview from Synapse
+        new_entityview_instance = await EntityView(id=entityview.id).get_async(
+            synapse_client=self.syn, include_columns=True
+        )
+        assert new_entityview_instance is not None
+        assert new_entityview_instance.name == entityview_name
+        assert new_entityview_instance.id == entityview.id
+        assert new_entityview_instance.description == entityview_description
+        assert new_entityview_instance.columns["test_column"].name == "test_column"
+        assert (
+            new_entityview_instance.columns["test_column"].column_type
+            == ColumnType.STRING
+        )
+        assert len(new_entityview_instance.columns) == 1
+
+    async def test_create_entityview_with_multiple_columns(
+        self, project_model: Project
+    ) -> None:
+        # GIVEN a entityview with multiple columns
+        entityview_name = str(uuid.uuid4())
+        entityview_description = "Test entityview"
+        entityview = EntityView(
+            name=entityview_name,
+            parent_id=project_model.id,
+            description=entityview_description,
+            columns=[
+                Column(name="test_column", column_type=ColumnType.STRING),
+                Column(name="test_column2", column_type=ColumnType.INTEGER),
+            ],
+            view_type_mask=ViewTypeMask.FILE,
+            include_default_columns=False,
+        )
+
+        # WHEN I store the entityview
+        entityview = await entityview.store_async(synapse_client=self.syn)
+        self.schedule_for_cleanup(entityview.id)
+
+        # THEN the entityview should be created
+        assert entityview.id is not None
+
+        # AND I can retrieve that entityview from Synapse
+        new_entityview_instance = await EntityView(id=entityview.id).get_async(
+            synapse_client=self.syn, include_columns=True
+        )
+        assert new_entityview_instance is not None
+        assert new_entityview_instance.name == entityview_name
+        assert new_entityview_instance.id == entityview.id
+        assert new_entityview_instance.description == entityview_description
+        assert new_entityview_instance.columns["test_column"].name == "test_column"
+        assert (
+            new_entityview_instance.columns["test_column"].column_type
+            == ColumnType.STRING
+        )
+        assert new_entityview_instance.columns["test_column2"].name == "test_column2"
+        assert (
+            new_entityview_instance.columns["test_column2"].column_type
+            == ColumnType.INTEGER
+        )
+        assert len(new_entityview_instance.columns) == 2
+
+    async def test_create_entityview_with_invalid_column(
+        self, project_model: Project
+    ) -> None:
+        # GIVEN a entityview with an invalid column
```

Comment:
I may have missed this, but what happens when an entity view is created with the default columns specified?